source/utils/config.json

```diff
@@ -1,6 +1,8 @@
 {
-  "emoji": "YYou are the author of the commit message. Your task is to select the appropriate category for the git diff based on the changes. Use the following categories (emoji category name => usage): 📦 new => for new files or new features; ✨ tweak => for enhancements or updates to the codebase; ☕ chore => for updates or changes outside the project codebase, including README.md; 🐞 fix => for fixing code bugs and errors. Please reply with the category name only.",
-  "message": "You are the author of the changes, you are going to provide a professional git commit message that is no longer than 25 characters in imperative present tense. Stricly no emojis are allowed and no conventional commit message as prefix is already provided. For example, instead of 'fix: fix a bug' make it 'fix a bug'. The message should be in lower case and no period at the end.",
+  "emoji": "You are the author of the commit message. Your task is to select the appropriate category for the git diff based on the changes. Use the following categories (emoji category name => usage): 📦 new => for new files or new features; ✨ tweak => for enhancements or updates to the codebase; ☕ chore => for updates or changes outside the project codebase, including README.md; 🐞 fix => for fixing code bugs and errors. Reply with the category in the format `<emoji> category (context)`. For example, `📦 new (adding feature)`.",
```

Comment:
AI will be confused here, because you introduced a new format. Without explaining what to input in (context) I guess we are using wrong term here. It should be "scope" instead of "context"

Good example here is like this ✨️ tweak (index.js): update logic

For modfying the output message look for the `openai.js` file you can find the return message before recommending it by the tool.

Reply:
Hey @Abhinav-Bansal751, can you please revert it to the original configuration? Since you added the latest changes to another file, Thanks!

Reply:
Kindly revert the original config @Abhinav-Bansal751 thanks!